packages/pn-pa-webapp/src/redux/auth/__test__/reducers.test.ts

```diff
@@ -93,39 +95,37 @@ describe('Auth redux state tests', () => {
       isFirstAccept: true,
       consentVersion: 'mocked-version',
     };
-    const apiSpy = jest.spyOn(ConsentsApi, 'getConsentByType');
-    apiSpy.mockResolvedValue(tosMock);
+    mockApi(apiClient, 'GET', GET_CONSENTS(ConsentType.TOS), 200, undefined, tosMock);
     const action = await store.dispatch(getToSApproval());
     const payload = action.payload as Consent;
     expect(action.type).toBe('getToSApproval/fulfilled');
     expect(payload).toEqual(tosMock);
-    apiSpy.mockRestore();
   });
 
   it('Should NOT be able to fetch the tos approval', async () => {
-    const tosErrorResponse = { response: { data: 'error-tos', status: 500 }};
-    const apiSpy = jest.spyOn(ConsentsApi, 'getConsentByType');
-    apiSpy.mockRejectedValue(tosErrorResponse);
+    const tosErrorResponse = { response: { data: 'error-tos', status: 500 } };
+    mockApi(apiClient, 'GET', GET_CONSENTS(ConsentType.TOS), 500, undefined, 'error-tos');
```

Comment:
here and above, we are sure we are using the correct responseBody? the api returns 'success' or is it axios?